middle_end/flambda2/to_cmm/to_cmm_env.ml

```diff
@@ -326,6 +326,10 @@ let inline_variable env var =
       if not (Variable.equal var var_from_stage)
       then will_not_inline_var env var
       else if binding.may_inline
+              &&
+              match consider_inlining_effectful_expressions with
+              | Some consider -> consider
+              | None -> Flambda_features.Expert.inline_effects_in_cmm ()
```

Comment:
maybe that match could be lifted out of the condition for the if-then-else

Reply:
Fixed